std/algorithm/iteration.d

```diff
@@ -788,24 +801,6 @@ Params:
     pred = predicate to apply to each element of the range
     r = range or iterable over which each iterates
 
-Example:
----
-void deleteOldBackups()
-{
-    import std.algorithm, std.datetime, std.file;
-    auto cutoff = Clock.currTime() - 7.days;
-    dirEntries("", "*~", SpanMode.depth)
-        .filter!(de => de.timeLastModified < cutoff)
-        .each!remove();
-}
----
```

Comment:
This code sample does not compile, and the other examples are much better at explaining its uses anyway so I didn't try to fix it.